Regressed by https://github.com/unicode-org/icu/commit/633438f8da99

In file included from breakiterator/breakiterator_class.cpp:21:
breakiterator/codepointiterator_internal.h:40:17: error: virtual function 'operator==' has a different return type ('UBool' (aka 'signed char')) than the function it overrides (which has return type 'bool')
                virtual UBool operator==(const BreakIterator& that) const;
                        ~~~~~ ^
/usr/local/include/unicode/brkiter.h:127:18: note: overridden virtual function is here
    virtual bool operator==(const BreakIterator&) const = 0;
            ~~~~ ^

--- breakiterator/codepointiterator_internal.cpp.orig	2021-09-21 17:07:44 UTC
+++ breakiterator/codepointiterator_internal.cpp
@@ -73,7 +73,11 @@ CodePointBreakIterator::~CodePointBreakIterator()
 	clearCurrentCharIter();
 }
 
+#if U_ICU_VERSION_MAJOR_NUM < 70
 UBool CodePointBreakIterator::operator==(const BreakIterator& that) const
+#else
+bool CodePointBreakIterator::operator==(const BreakIterator& that) const
+#endif
 {
 	if (typeid(*this) != typeid(that)) {
 		return false;
--- breakiterator/codepointiterator_internal.h.orig	2021-09-21 17:07:44 UTC
+++ breakiterator/codepointiterator_internal.h
@@ -37,7 +37,11 @@ namespace PHP {
 
 		virtual ~CodePointBreakIterator();
 
+#if U_ICU_VERSION_MAJOR_NUM < 70
 		virtual UBool operator==(const BreakIterator& that) const;
+#else
+		virtual bool operator==(const BreakIterator& that) const;
+#endif
 
 		virtual CodePointBreakIterator* clone(void) const;
 
